### PR TITLE
Update the desciption of the developer setup

### PR DESCRIPTION
**Pull request by cmnrd**
The current developer setup description is outdated and also contained several errors. Also there were different descriptions in the handbook and on the Download page. This PR fixes and updates the description in the handbook, and simply links to the handbook article from the download page.

### CODE REVIEW

File: packages/documentation/copy/en/developer/Downloading and Building.md [hunk 1]
```diff
@@ -1,34 +1,47 @@
...
-If instead you want to use the **most recent development version** of Lingua Franca, then you will be given the option to download the Epoch IDE or the command-line tools from the nightly build. Just follow the above links.
-The Vim plugin provides syntax-directed editing, but it does not include the diagram synthesis that is available with Epoch and Code.
-The diagram synthesis is quite useful, so we recommend using Epoch or Code to develop LF programs.
+## Command Line Tools
 
-## The Repository
+The command line tools are used using Gradle with `./gradlew buildAll`.
 
-If you plan to contribute to Lingua Franca, or if you want to keep up to date as the project evolves, you will need to work from the git repository on GitHub. There are several ways to do this:
+You can the Lingua Franca compiler and other CLI tools. For instance: `./bin/lfc --version`.
```
Comment (edwardalee) [inline packages/documentation/copy/en/developer/Downloading and Building.md:23]: This sentence is incomplete. Not sure what it intends to say.  Should we list the command line tools?

File: packages/documentation/copy/en/developer/Downloading and Building.md [hunk 2]
```diff
@@ -1,34 +1,47 @@
...
 
-1. [IntelliJ setup](/docs/handbook/intellij) (Recommended if you plan to do Kotlin development)
-2. [Oomph setup for Eclipse](/docs/handbook/eclipse-oomph) (This does not currently support Kotlin development, used in C++ and Rust code generators)
-3. [Clone the Repository](https://github.com/lf-lang/lingua-franca) and build manually using Gradle or Maven:
+## Epoch
 
-   - Gradle: `./gradlew assemble` (the `build` also performs tests, which takes a long time)
-   - Maven: `mvn compile` (you need to install Maven first)
+The Epoch IDE can be build with Maven using `mvn compile && mvn pacakge`. The resulting is then located within `org.lflang.rca/target/products/org.lflang.rca/`. On a 64-bit Linux, the binary is `org.lflang.rca/target/products/org.lflang.rca/linux/gtk/x86_64/epoch/epoch`.
```
Comment (edwardalee) [inline packages/documentation/copy/en/developer/Downloading and Building.md:27]: The resulting what? App?

**Review by edwardalee — CHANGES_REQUESTED** [review]
Looks good!  A couple of incomplete sentences need fixing.

**Comment by cmnrd** [comment]
Thanks for finding those problems! I pushed a fix

File: packages/documentation/copy/en/developer/Downloading and Building.md [hunk 3]
```diff
@@ -1,34 +1,47 @@
 ---
-title: Downloading
+title: Developer Setup
```
Comment (lhstrh) [inline packages/documentation/copy/en/developer/Downloading and Building.md:2]: The title here does not match the file name anymore...

Comment (cmnrd) [reply packages/documentation/copy/en/developer/Downloading and Building.md]: I was a bit of afraid renaming the file as I did not know how it is included in the build and found, but apparently a simple file rename just works.

Comment (cmnrd) [reply packages/documentation/copy/en/developer/Downloading and Building.md]: Or not... I don't know why the CI build fails now, it works fine locally

**Comment by lhstrh** [comment]
It says:
```
Error: Could not find the file 'developer/Downloading and Building.md' from the handbook nav
```

**Comment by cmnrd** [comment]
Yes, but I don't know what that means. It works fine locally. I guess a reference needs to be updated somewhere, but I have no clue where... The only files that I found referencing the file appear to be generated.

**Comment by lhstrh** [comment]
> Yes, but I don't know what that means. It works fine locally. I guess a reference needs to be updated somewhere, but I have no clue where... The only files that I found referencing the file appear to be generated.

Will look into it!

File: packages/documentation/copy/en/developer/Downloading and Building.md (hunk 2 above)
Comment (lhstrh) [inline packages/documentation/copy/en/developer/Downloading and Building.md]:
```suggestion
The Epoch IDE can be built with Maven using `mvn clean package`. The resulting tar an zip archives for all supported platforms can be found in `./org.lflang.rca/target/products/`. Platform-specific binaries can be found in the `org.lflang.rca` subdirectory. For instance, a 64-bit Linux binary will be located in `./org.lflang.rca/target/products/org.lflang.rca/linux/gtk/x86_64/epoch/epoch`.
```

File: packages/documentation/copy/en/developer/Developer Setup.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## Command Line Tools
+
+You can the build the Lingua Franca compiler and other CLI tools with Gradle using `./bin/lfc --version`.
+
+There are scripts in `bin` which you can use to run the compiler or other tools. For instance: `./bin/lfc --version`.
+
+## Epoch
+
+The Epoch IDE can be build with Maven using `mvn compile && mvn pacakge`. The resulting executable is then located within `org.lflang.rca/target/products/org.lflang.rca/`. On a 64-bit Linux, the binary is `org.lflang.rca/target/products/org.lflang.rca/linux/gtk/x86_64/epoch/epoch`.
```
Comment (lhstrh) [inline packages/documentation/copy/en/developer/Developer Setup.md:27]:
```suggestion
The Epoch IDE can be built with Maven using `mvn clean package`. The resulting tar an zip archives for all supported platforms can be found in `./org.lflang.rca/target/products/`. Platform-specific binaries can be found in the `org.lflang.rca` subdirectory. For instance, a 64-bit Linux binary will be located in `./org.lflang.rca/target/products/org.lflang.rca/linux/gtk/x86_64/epoch/epoch`.
```

**Comment by cmnrd** [comment]
@lhstrh did you have a look at the problem. I am inclined to simply revert the renaming and merge.

**Comment by lhstrh** [comment]
Are you sure that the link to the page that you created reflects the revert?

**Comment by cmnrd** [comment]
I am not sure what you mean, but yes all the links work fine and I double-checked them on the live version of the website. I don't understand at all why the name of the markdown file matters, as everything is generated from it, and it indicates its permalink with `permalink: /docs/handbook/developer-setup`.

Also locally everything works fine weather I rename the markdown file or not. So it seems like there might be something wrong with the CI setup.

**Pull request merged**